lua/neogit/popups/rebase.lua

```diff
@@ -10,6 +10,12 @@ function M.create()
   local p = popup
     .builder()
     :name("NeogitRebasePopup")
+    :action("r", "Continue rebase", function()
```

Comment:
The binding `c` isn't more intuitive ? c for continue.

Reply:
You can use `action_if` to only display these actions when there is rebase in progress ?

Reply:
I used `rr` as that is the binding in vim-fugitive, partly because it has the same double tap as `cc` for commit.

Reply:
I never used vim-fugitive so I was just thinking of a binding more intuitive for any users

Reply:
Alright, I've added action_if and skip as well

Reply:
Thank you for your work !